brainzutils/metrics.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import division
+
+import datetime
+from functools import wraps
+
+import six
+from redis import ResponseError
+
+from brainzutils import cache
+
+NAMESPACE_METRICS = "metrics"
+REDIS_MAX_INTEGER = 2**63-1
+STATS_COUNT_DATE_KEY = "date"
+
+_metrics_site_name = None
+
+
+def init(site):
```

Comment:
Some documentation on what site here is would be nice. Does it mean `project`?

Reply:
I think renaming this to project might make more sense, we might want to use this for non-website projects.